.golangci.yml

```diff
@@ -70,3 +70,6 @@ issues:
     - linters:
         - gosec
       text: "G601:" # G601: Implicit memory aliasing in for loop. (gosec)
+    - linters:
+        - gosec
+      text: "G112:" # G112: Potential Slowloris Attack because ReadHeaderTimeout is not configured in the http.Server (gosec)
```

Comment:
why ignore this rule? please add a code snippet which will cause this issue.

Reply:
added //nolint to skip this rule for that not configured `ReadHeaderTimeout` in http.Server

which will cause this issue is very clearly as text

Reply:
which will cause this issue added `//nolint` and TODO to skip this issue